internal/project/framework/internals/latency/latency.go

```diff
@@ -1,12 +1,13 @@
-{{ .Copyright }}
+/**#bean*/ /*#bean.replace({{ .Copyright }})**/
 package latency
 
 import (
 	"bytes"
 	"encoding/binary"
 	"time"
 
-	"{{ .PkgPath }}/framework/internals/global"
+	/**#bean*/
+	"demo/framework/internals/global" /*#bean.replace("{{ .PkgPath }}/framework/internals/global")**/
```

Comment:
put these two lines in one line

Reply:
`go fmt` command make it, `go fmt` will run before I commit,but I don't know the fmt rules
bean macro still works in two lines